easybuild/easyconfigs/n/numpy/numpy-1.8.2-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,26 @@
+name = 'numpy'
+version = '1.8.2'
+
+homepage = 'http://www.numpy.org'
+description = """NumPy is the fundamental package for scientific computing with Python. It contains among other things:
+ a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran
+ code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,
+ NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be 
+ defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['numpy-1.8.1-mkl.patch']
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
```

Comment:
@jrbosch for new easyconfig, please use the `%(pyver)s` template (supported since EasyBuild v2.7.0), which avoids the need for the 'local variables' `python` and `pyver`:

```python
name = 'numpy'
version = '1.8.2'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
]
```